docs/docs/css/main.css

```diff
@@ -93,6 +93,14 @@ iframe {
 	border: none;
 }
 
+img {
+	height: auto;
```

Comment:
Ensures images don't stretch (if `max-width: 100%` would cause an image to be smaller than its declared or intrinsic width).